cmake-scripts/Dependencies.cmake

```diff
@@ -0,0 +1,134 @@
+# Dependencies.cmake
+# this file locates all third party dependencies, and creates easy to use
+# targets like `openomf::argtable` for the openomf build scripts to link to.
+
+# set module path only temporarily, to discourage find_package being used outside of this script
+set(ORIGINAL_CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH})
+set(CMAKE_MODULE_PATH ${PROJECT_SOURCE_DIR}/cmake-scripts)
+
+
+# SDL2, SDL2_mixer
+if(VCPKG_TOOLCHAIN)
```

Comment:
Could these if-elses be minimized to just contain stuff that is different ? And move the identical lines outside of the if-else block ?

Reply:
Moved the add_library calls outside of the vcpkg_toolchain if/else's, thanks-- that helps make it clear what targets Dependencies.cmake is providing to the rest of the scripts.
Note that I kept the find_package calls intentionally within the if-else blocks.

Also fixed the mess of tabs and spaces that was the indentation in this file (oops)